test/e2e/traceflow_test.go

```diff
@@ -52,18 +51,9 @@ func TestTraceflow(t *testing.T) {
 	}
 
 	node1 := nodeName(0)
-	node2 := nodeName(1)
 
-	node1Pods, node1IPs, node1CleanupFn := createTestBusyboxPods(t, data, 2, node1)
-	node2Pods, node2IPs, node2CleanupFn := createTestBusyboxPods(t, data, 1, node2)
+	node1Pods, node1IPs, node1CleanupFn := createTestBusyboxPods(t, data, 3, node1)
```

Comment:
It seems you only use 2 Pods in the tests below, not 3 (Pod with index 0 and Pod with index 2)

I also notice that you don't have a test case for denied traffic because of a NetworkPolicy?

Reply:
Ignore this. I misread the diff because how it was displayed in Github